packages/preset-mini/src/_rules/gap.ts

```diff
@@ -20,4 +22,5 @@ function handleGap([, d = '', s]: string[], { theme }: RuleContext<Theme>) {
 export const gaps: Rule[] = [
   [/^(?:flex-|grid-)?gap-?()(.+)$/, handleGap, { autocomplete: ['gap-$spacing', 'gap-<num>'] }],
   [/^(?:flex-|grid-)?gap-([xy])-?(.+)$/, handleGap, { autocomplete: ['gap-(x|y)-$spacing', 'gap-(x|y)-<num>'] }],
```

Comment:
I think the use of x/y should be marked as deprecated

Reply:
I don't have any opinion. There has been no such discussion yet, and either way is fine, provided that you can tell me how to handle deprecation...

Reply:
I am thinking using `warnOnce` with adjusted message

https://github.com/unocss/unocss/blob/0d463d0cc7e78eaf28c1121c94bb8adaa8b72800/packages/preset-mini/src/_rules/container.ts#L5-L6

Reply:
If it's still used in Tailwind, I guess we can't just deprecate it. For ppl that understand the behaviour of `gap-x|y`, I guess it's fine to keep using them (I don't think ppl would have enough incentive to migrate existing projects as well)

Reply:
@chu121su12 So, is it okay to go as it is?

Reply:
Yes. LGTM